### ISSUE TITLE
Clarify what "Download" means for metadata only entries

### ISSUE THREAD

**Issue by datadavev** [NCEAS/metacatui#249]
Some users are finding the download link in search results to be confusing when the entry is a metadata only record. They expect the Download to download the data package, even though of course this is not possible when the data is not registered with DataONE.

It would be worth evaluating how the UI could be adjusted to make it clearer to users what Download means for metadata only entries.

**Comment by csjx** [comment]
Agreed. While I think it's helpful to be able to get to the raw XML in some way, it's very prominent in the UI and I've always thought it could be de-emphasized.  Perhaps it's shown as just a different view of the web page rendering like is done at GCIS.
